src/views/components-default/TheDefaultMobileNavigation.vue

```diff
@@ -1,100 +1,92 @@
 <template>
   <div class="mew-component--landing-page-menu-mobile">
-    <mew-button
-      class="mobile-menu-button ml-n3"
-      color-theme="white"
-      btn-style="transparent"
-      icon="mdi-text"
-      icon-type="mdi"
-      icon-align="left"
-      style="border-radius: 100% !important; padding: 0"
-      @click.native="isOpen = true"
-    />
+    <app-btn-menu height="40" x-large @click.native="isOpen = true" />
```

Comment:
If you want we can make default size to be 40px and x-large. in that way the icon will stay consistent through out all nav drawers. Since it's only used in 2 places. 

Also AppBtnMenu has a prop function for menuMethod. In stead of this:   @click.native="isOpen = true" you can use menu-method="isOpen = true". Otherswise if you want to keep it @click.native then can you change another drawer to it and remove the prop menu-method from the AppBtnMenu. This will make it consistent accross the implementations.
```
@click.native="setIsOpen"

setIsOpen() {
 this.isOpen = true
}
```

Reply:
fixed